types/shippo/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6"
+        ],
+        "esModuleInterop": true,
```

Comment:
This should never be enabled in DefinitelyTyped

Reply:
How do I have a default export without this? @andrewbranch 

![image](https://user-images.githubusercontent.com/2819256/103710616-13ff4400-4f7b-11eb-8f68-4a0f93744738.png)

Reply:
You don’t, because the library does not have a default export. The correct import style is `import Shippo = require("shippo")`.

Reply:
I don't fully understand how the `import` syntax and the `require` syntax work together. I assumed one was meant to replace the other.

Reply:
(Explained in some detail [here](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/50413#discussion_r552266602))